src/core/templates-manager.ts

```diff
@@ -78,7 +78,14 @@ class TemplatesManager {
                     element.classList.add(DX_TEMPLATE_WRAPPER_CLASS);
                 }
 
-                events.one(element, DX_REMOVE_EVENT, mountedTemplate.$destroy.bind(mountedTemplate));
+                if (element.nodeType === 3) {
```

Comment:
I suggest extract it to a constant
```suggestion
                if (element.nodeType === TEXT_NODE) {
```